apps/meteor/tests/end-to-end/api/rooms.ts

```diff
@@ -2290,6 +2300,58 @@ describe('[Rooms]', () => {
 					expect(res.body.room).to.not.have.property('favorite');
 				});
 		});
+		it('should update the team sidepanel items to channels and discussions', async () => {
+			const sidepanelItems = ['channels', 'discussions'];
+			const response = await request
+				.post(api('rooms.saveRoomSettings'))
+				.set(credentials)
+				.send({
+					rid: testTeam.roomId,
+					sidepanel: { items: sidepanelItems },
+				});
+
+			expect('Content-Type', 'application/json');
+			expect(200);
+			expect(response.body).to.have.property('success', true);
+
+			await request
+				.get(api('channels.info'))
+				.set(credentials)
+				.query({ roomId: response.body.rid })
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((response) => {
+					expect(response.body).to.have.property('success', true);
+					expect(response.body.channel).to.have.property('sidepanel');
+					expect(response.body.channel.sidepanel).to.have.property('items').that.is.an('array').to.have.deep.members(sidepanelItems);
+				});
+		});
+		it('should throw error when updating team sidepanel with incorrect items', async () => {
+			const sidepanelItems = ['wrong'];
+			await request
+				.post(api('rooms.saveRoomSettings'))
+				.set(credentials)
+				.send({
+					rid: testTeam.roomId,
+					sidepanel: { items: sidepanelItems },
+				});
+
+			expect('Content-Type', 'application/json');
+			expect(400);
+		});
+		it('should throw error when updating team sidepanel more than 2 items', async () => {
```

Comment:
What if duplicated values are provided? Should we accept that? :thinking:

Reply:
No we shouldn't accept duplicated items. I'll add a test